### ISSUE TITLE
Value of train-loss and valid-loss acceptable to use?

### ISSUE THREAD

**Issue by huyvt811** [DuncanZauss/openpifpaf_mpii#1]
Hello @DuncanZauss 
Thank you for created a plugin support training pose estimation PifPaf with MPII dataset. But I have some questions. Hope you explain and give me some advice. 

1. When I train the model, I see the return loss at some batch is negative. Is that normal, or do I have an overfit or something else problem?
2. Value of train-loss and valid-loss acceptable to use?

**Comment by DuncanZauss** [comment]
Hi @huyvt811,
Thanks for your interest in this plugin!
1) A negative loss is expected, this is no problem.
2) I don't quite understand your question, but both train and val loss should be printed during training and if the val loss is much higher than the train loss you are overfitting, otherwise everything should be okay.

**Comment by huyvt811** [comment]
Hi @DuncanZauss ,
Sorry, my English is not good so i didn't express it properly.

> 2. I don't quite understand your question, but both train and val loss should be printed during training and if the val loss is much higher than the train loss you are overfitting, otherwise everything should be okay.

When I trained model Openpifpaf, I see in log has info
{"message": "", "levelname": "INFO", "name": "openpifpaf.network.trainer", "asctime": "2023-04-18 10:05:10,638", "type": "train-epoch", "epoch": 400, "loss": 218.90409, "head_losses": [-242.49308, 197.50375, 0.03971, -274.79174, 538.55699, 0.08846], "time": 895.0, "n_clipped_grad": 0, "max_norm": 0.0}
{"message": "", "levelname": "INFO", "name": "openpifpaf.network.trainer", "asctime": "2023-04-18 10:20:19,177", "type": "val-epoch", "epoch": 400, "loss": 196.12896, "head_losses": [-239.38579, 186.12606, 0.03856, -275.3778, 524.6394, 0.08852], "time": 908.5}
Ex, at epochs 400, loss 218.90409. I wonder how much loss is small and I can stop training and use it. It's seem like value loss of train and val still decreases.
